src/ctap/storage.rs

```diff
@@ -420,8 +420,7 @@ impl From<StoreError> for Ctap2StatusCode {
 }
 
 fn deserialize_credential(data: &[u8]) -> Option<PublicKeyCredentialSource> {
-    let cbor = cbor::read(data).ok()?;
-    cbor.try_into().ok()
+    PublicKeyCredentialSource::try_from(&cbor::read(data).ok()?).ok()
```

Comment:
What is the rationale for changing this test?

Reply:
This is because when `TryFrom` is on a reference, the `.try_into()` notation doesn't take the reference implicitly, so we would either need to write `(&x).try_into()` or `T::try_from(&x)`. I chose the second to avoid ping-pong due to the possibly confusing `(&x).try_into()` notation.